tests/Common/SpanIntegrationChecker.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace DDTrace\Tests\Common;
+
+use DDTrace\Contracts\Span;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Checks that spans are assigned the proper integration.
+ */
+class SpanIntegrationChecker
+{
+    /**
+     * Returns the known matching patterns operation ==> integration class.
+     */
+    public function defineIntegrationsByPattern()
```

Comment:
This can be static and private. :)
```suggestion
    private static function defineIntegrationsByPattern()
```

Reply:
If you don't mind (no problem on changing my mind) I'd prefer to keep this classes with non-static methods unless we have to do it for functional/performance reasons (and of course this is not the case for testing purposes).

The point is that it happened more often that I had to inject a service at a certain point than I needed to access a class when I could not create an instance for performance reasons 😄 

Again totally opened to do this if you don't agree.